veg2hab/io/cli.py

```diff
@@ -40,6 +40,9 @@ def _decorate_click(func: Callable, param_schema: Dict):
         if field_info.get("format", "") == "path":
             writable = field_name == "output"
             param_type = click.Path(exists=False, writable=writable)
+        elif field_name == "welke_typologie":
+            ref_name = field_info.get("allOf")[0].get("$ref").split("/")[-1]
+            param_type = click.Choice(param_schema["definitions"][ref_name]["enum"])
```

Comment:
Dit gaat me echt boven de pet, maar als jij zegt dat t nodig is vind ik t goed

Reply:
Ja ik ben hier een json bestandje uit aan het pakken. Het is niet zo mooi. Misschien moet ik 'm ook maar gewoon hardcoden..